ci-operator/step-registry/storage/create/csi-aws-efs/sts-operator-role/storage-create-csi-aws-efs-sts-operator-role-ref.metadata.json

```diff
@@ -0,0 +1,11 @@
+{
+	"path": "storage/create/csi-aws-efs/sts-operator-role/storage-create-csi-aws-efs-sts-operator-role-ref.yaml",
+	"owners": {
+		"approvers": [
+			"storage-approvers"
+		],
+		"reviewers": [
+			"storage-reviewers"
+		]
+	}
+}
```

Comment:
Missing newline.

Reply:
Addressed.

Reply:
It seems the `ci/prow/step-registry-metadata` check does not allow the newline to dismiss the warning. I just use the `make registry-metadata` to generate these metadata json files, it generated without newline.